doc/hardware/arch/arm-scmi.rst

```diff
@@ -139,6 +139,8 @@ Currently, Zephyr has support for the following standard protocols:
 	#. **Clock management**
 	#. **Pin Control**
 
+NXP-specific protocols:
```

Comment:
Maybe it would be good to have a separate documentation page for each vendor? Also, perhaps it would be useful for the community to document the vendor protocols? Like Linux did here: https://elixir.bootlin.com/linux/v6.13.7/source/drivers/firmware/arm_scmi/vendors/imx/imx95.rst.

CC: @kartben any takes on this?

Reply:
Hi Laurentiu, Thanks for your comments, they are very helpful. Will fix in next version, I also have some comments, would you please review?